tests/conftest.py

```diff
@@ -0,0 +1,107 @@
+import os
+from fastapi.testclient import TestClient
+import pytest
+
+
+class AuthClient:
+    def __init__(self, client, base_url):
+        self._client = client
+        self._base_url = base_url
+
+    def login(self, username="abcd", password="abcd"):
+        res = self._client.post(
+            f"{self._base_url}/auth/login",
+            json={"username": username, "password": password, "plugin": "dummy"},
+        )
+
+        assert res.status_code == 201
+
+        self._token = res.json()["token"]
+        return res
+
+    def logout(self):
+        return
+        # TODO: no logout method yet!
```

Comment:
Well actually we don't need one since the auth token is not stored anywhere on the server side. Once the client discards the token, it does not exist anywhere anymore. Therefore, the user is logged out.
The server is never aware of which users are currently logged in, so it can't have any logout route.

Reply:
I guess this will go away in the future then:
https://github.com/ispyb/py-ispyb/blob/80fb2ec3bd467691a625ed3291d9e0920bd2be61/pyispyb/app/routes/auth.py#L81 ?

Reply:
Oh, yes. This was an experimentation to create a database token, which is used by the **Java** ISPyB backend. py-ispyb just creates it and then never uses it.
With this, we were able to log-in into py-ispyb and then use both backends simultaneously without having to handle two different logging in.
But then it was decided to use legacy routes in py-ispyb instead, so I think it could be removed!

Reply:
I've removed the login method from the `auth_client`